<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2504</width>
    <height>1154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonPlane">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Projection</string>
    </property>
   </widget>
   <widget class="QLabel" name="isometric">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string>isometric</string>
    </property>
   </widget>
   <widget class="QLabel" name="planeProj">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>plane projection</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Ax">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="By">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Cz">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Dd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="orthographic">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>70</y>
      <width>661</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string>orthographic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUP">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDOWN">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>720</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLEFT">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>235</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRIGHT">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonZplus">
    <property name="geometry">
     <rect>
      <x>1235</x>
      <y>30</y>
      <width>41</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Z+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonZminus">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>30</y>
      <width>41</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2504</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
